front/app/components/PageMeta/index.tsx

```diff
@@ -0,0 +1,73 @@
+// libraries
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { adopt } from 'react-adopt';
+
+// i18n
+import { injectIntl, MessageDescriptor } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+
+// resources
+import GetAuthUser, { GetAuthUserChildProps } from 'resources/GetAuthUser';
+import GetAppConfigurationLocales, {
+  GetAppConfigurationLocalesChildProps,
+} from 'resources/GetAppConfigurationLocales';
+
+// utils
+import getAlternateLinks from 'utils/cl-router/getAlternateLinks';
+import getCanonicalLink from 'utils/cl-router/getCanonicalLink';
+
+interface InputProps {
+  titleMessage: MessageDescriptor;
+  descriptionMessage: MessageDescriptor;
+}
+
+interface DataProps {
+  authUser: GetAuthUserChildProps;
+  tenantLocales: GetAppConfigurationLocalesChildProps;
+}
+
+interface Props extends InputProps, DataProps {}
+
+const IdeaMeta = React.memo<Props & InjectedIntlProps>(
+  ({ intl, authUser, tenantLocales, titleMessage, descriptionMessage }) => {
+    const { formatMessage } = intl;
+    const { location } = window;
+    const ideasIndexTitle = formatMessage(titleMessage);
+    const ideasIndexDescription = formatMessage(descriptionMessage);
+
+    return (
+      <Helmet>
+        <title>
+          {`
+          ${
+            authUser && authUser.attributes.unread_notifications
+              ? `(${authUser.attributes.unread_notifications}) `
+              : ''
+          }
+          ${ideasIndexTitle}`}
+        </title>
+        {getAlternateLinks(tenantLocales)}
+        {getCanonicalLink()}
+        <meta name="title" content={ideasIndexTitle} />
+        <meta name="description" content={ideasIndexDescription} />
+        <meta property="og:title" content={ideasIndexTitle} />
+        <meta property="og:description" content={ideasIndexDescription} />
+        <meta property="og:url" content={location.href} />
+      </Helmet>
+    );
+  }
+);
+
+const IdeaMetaWithHoc = injectIntl<Props>(IdeaMeta);
```

Comment:
Given you are creating a new component, I would use hooks instead of resource components.